app/__init__.py

```diff
@@ -6,6 +6,25 @@
 app = Flask(__name__)
 
 
+
 @app.route('/')
 def index():
     return render_template('index.html', title="MLH Fellow", url=os.getenv("URL"))
+
+
+@app.route('/work-Josh')
+def workJosh():
+    return render_template('work-experience-Josh.html', title="Work Experience", url=os.getenv("URL"))
+
+@app.route('/work-Juli')
+def workJuli():
+    return render_template('work-experience-Juli.html', title="Work Experience", url=os.getenv("URL"))
+
+
+@app.route('/work-Maansi')
+def workMaansi():
+    return render_template('work-experience-Maansi.html', title="Work Experience", url=os.getenv("URL"))
+
+
```

Comment:
also can you add the block below to it

@app.route('/jul-travel')
def julTravel():
    return render_template('/juliette/jul-travel.html', title="MLH Fellow", url=os.getenv("URL"))

this is a part i had wrote and merged to our main already, so adding this to your file will just make our merging a little easier, since right now there is a conflict because yours does not have this